test/tests/rails5.rb

```diff
@@ -206,4 +206,16 @@ def test_cross_site_scripting_strip_tags_cve
       :relative_path => "Gemfile.lock",
       :user_input => nil
   end
+
+  def test_dangerous_eval_in_prior_class_method_with_same_name
+    assert_warning :type => :warning,
+      :warning_code => 13,
+      :fingerprint => "7fe3142d1d11b7118463e45a82b4b7a2b5b5bac95cf8904050c101fae16b8168",
+      :warning_type => "Dangerous Eval",
+      :line => 3,
+      :message => /User input in eval near line 3/,
+      :confidence => 0,
+      :relative_path => "app/models/user.rb",
+      :user_input => s(:params)
```

Comment:
Fixed! I had noted that you had asked for that above, but then completely forgot :blush: